include/dxc/Support/dxcapi.use.h

```diff
@@ -116,8 +117,8 @@ class DxcDllSupport {
                            (IUnknown **)pResult);
   }
 
-  HRESULT CreateInstance2(IMalloc *pMalloc, REFCLSID clsid, REFIID riid,
-                          IUnknown **pResult) {
+  virtual HRESULT CreateInstance2(IMalloc *pMalloc, REFCLSID clsid, REFIID riid,
+                                  IUnknown **pResult) {
```

Comment:
In this change, I think we should also be overriding `CreateInstance` and `CreateInstance2` in `DxcDllExtValidationSupport` to redirect through to one of the `DxilSupport` create methods when `clsid == `CLSID_DxcValidator`.